pkg/kubectl/cmd/util/factory.go

```diff
@@ -248,7 +243,7 @@ func GetFirstPod(client coreclient.PodsGetter, namespace string, selector string
 	if err != nil {
 		return nil, 0, err
 	}
-	pods := []*v1.Pod{}
+	var pods []*v1.Pod
```

Comment:
I really dislike this construction.  Is it an editor problem?

Reply:
editor dislikes them

Reply:
What kind of editor you're using :stuck_out_tongue_closed_eyes: